Prettier

enginmertcan opened the pull request.

enginmertcan committed:
feat(config): TN-2 TN-4 Update ESLint and Prettier configurations

- Added new ESLint rules to .eslintrc.json to enhance code quality (TN-4).
- Configured prettier.config.mjs with @ianvs/prettier-plugin-sort-imports to handle import sorting (TN-2).

This commit addresses both tasks related to code formatting and linting.

servercetinbe commented on package.json:
```diff
@@ -6,19 +6,37 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
-    "lint": "next lint"
```
Eeslint ve prettier icin eksik paketleri indirmeliyiz

enginmertcan committed:
fix(eslint): TN-2 TN-4 eslint and prettier  missing dependencies fixed

Missing Prettier and ESLint dependencies were installed and configuration issues were resolved. Now both Prettier and ESLint are functioning correctly.
